app/state_machines/efile_submission_state_machine.rb

```diff
@@ -120,21 +120,26 @@ class EfileSubmissionStateMachine
     tax_return.transition_to(:file_accepted)
 
     benefits = Efile::BenefitsEligibility.new(tax_return: tax_return, dependents: submission.qualifying_dependents)
-    ctc_amount = benefits.ctc_amount / 2
-    refund_amount = client.intake.claim_owed_stimulus_money_no? ? 0 : (benefits.eip1_amount + benefits.eip2_amount)
-    eip3_amount = benefits.eip3_amount
+    eip1_and_eip2_amount = [benefits.eip1_amount, benefits.eip2_amount].compact.sum
```

Comment:
There's a lot of calculations happening here. I wonder if its a good time time to move these calculations into a method on accepted_tax_return_analytics where we can make the analytics object based on the tax return object passed in so that we have a separation of concerns, especially now that we have more cals and data here.